week1/week1-assigment/index.js

```diff
@@ -0,0 +1,52 @@
+// To run this file: node index.js
+// Then in your browser: http://localhost:8080
+var http = require('http');
+
+var port = 8080;
```

Comment:
since these things don't change during the running of the application, `const` would be better than `var` or `let`. You should review the differences between `var`, `let`, and `const`.

Reply:
i knw to difference bt i thought like we should not change those few lines

Reply:
Okay, that's understandable - I just wanted to make sure you were aware. We'll look at changing it in the original, thanks for pointing that out.